pokemongo_bot/event_handlers/social_handler.py

```diff
@@ -83,7 +83,11 @@ def initialize(self):
             return
 
     def run(self):
-        self._mqttc.connect("broker.pikabot.org", 1883, 20)
+        try:
+            self._mqttc.connect("broker.pikabot.org", 1883, 20)
+        except Exception as e:
```

Comment:
I don't remember the exact type, but you get an exception if your firewall doesn't allow outgoing traffic on port 1883, crashing the application